keyboards/gergo/keymaps/germ/readme.md

```diff
@@ -0,0 +1,26 @@
+# Gergo
```

Comment:
This file should be moved to and combined with the existing `keyboards/gergo/readme.md`. As this information is general, it should be at the keyboard level.

Keymap-level readme files should be specific to the keymap they're included with.